soccer/gameplay/tactics/defense.py

```diff
@@ -61,7 +76,21 @@ def debug(self):
     def debug(self, value):
         self._debug = value
 
-    def execute_running(self):
+    def execute_defending(self):
+
+        defender1 = self.subbehavior_with_name('defender1')
+        defender2 = self.subbehavior_with_name('defender2')
+        close_defender = defender1
+        if (main.ball().pos.mag() < constants.Field.ArcRadius * 2):
+            if (defender1 != None and defender2 != None):
```

Comment:
Ah, so the `if (defender1.robot != None and defender2.robot != None):` should take of what we want (logically). Thanks!